src/types/julia.jl

```diff
@@ -139,6 +139,7 @@ function save_democards(card_dir::String,
                         src="src",
                         throw_error = false,
                         properties = Dict{String, Any}(),
+                        literate_kwargs = (),
```

Comment:
Should it be NamedTuple instead of Tuple?

```suggestion
                        literate_kwargs::NamedTuple = (;),
```